crates/npmrc/src/custom_deserializer.rs

```diff
@@ -127,4 +161,22 @@ mod tests {
         assert_eq!(store_dir, Path::new("/tmp/xdg_data_home/pacquet/store"));
         env::remove_var("XDG_DATA_HOME");
     }
+
+    #[cfg(windows)]
+    #[test]
+    fn test_should_get_the_correct_drive_letter() {
+        let current_dir = Path::new("C:\\Users\\user\\project");
+        let drive_letter = get_drive_letter(current_dir);
+        assert_eq!(drive_letter, Some('C'));
+    }
+
+    #[cfg(windows)]
+    #[test]
+    fn test_default_store_dir_with_windows() {
+        let current_dir = Path::new("D:\\Users\\user\\project");
+        let home_dir = Path::new("C:\\Users\\user");
+
+        let store_dir = default_store_dir_windows(&home_dir, &current_dir);
+        assert_eq!(store_dir, Path::new("D:\\.pacquet-store"));
+    }
```

Comment:
i think we need one more test for when the home dir is on the same drive as the current dir.